qualification/tied_array_channelised_voltage/test_delay.py

```diff
@@ -21,24 +21,12 @@
 from matplotlib.figure import Figure
 from pytest_check import check
 
-from katgpucbf import DIG_SAMPLE_BITS
 from katgpucbf.fgpu.delay import wrap_angle
 
 from .. import CBFRemoteControl, TiedArrayChannelisedVoltageReceiver
 from ..reporter import POTLocator, Reporter
 
 
-async def dsim_gaussian(cbf: CBFRemoteControl, pdf_report: Reporter, amplitude: float) -> None:
-    """Configure the dsim with Gaussian noise."""
-    pdf_report.step("Configure the D-sim with Gaussian noise.")
-    dig_max = 2 ** (DIG_SAMPLE_BITS - 1) - 1
-    # Small so that we don't saturate, as delaying and undelaying a saturated
-    # value won't round-trip properly
```

Comment:
Did you purposefully remove this comment/note *during the move? I thought it seemed quite useful - maybe your reasoning changed?

EDIT: Although looking at the usages later on, I see the same or similar comment above each call, so 🤷

Reply:
Yes, it shouldn't have been in the function in the first place, because within the function the amplitude could be small or large. It's the call sites where a small amplitude is chosen.